statsmodels/discrete/count_model.py

```diff
@@ -170,9 +170,6 @@ def fit(self, start_params=None, method='bfgs', maxiter=35,
             full_output=1, disp=1, callback=None,
             cov_type='nonrobust', cov_kwds=None, use_t=None, **kwargs):
         if start_params is None:
-            offset = getattr(self, "offset", 0) + getattr(self, "exposure", 0)
-            if np.size(offset) == 1 and offset == 0:
-                offset = None
```

Comment:
Here and below these were apparently copy/pasted from discrete_model, are not actually necessary.